src/hera/task.py

```diff
@@ -522,6 +527,7 @@ def get_script_def(self) -> V1alpha1ScriptTemplate:
             resources=self.argo_resources,
             volume_mounts=self.get_volume_mounts(),
             working_dir=self.working_dir,
+            security_context=V1SecurityContext(privileged=self.privileged),
```

Comment:
What do you think about isolating this into a class method? I think there is some interest in extending the usage of security contexts so that FS groups can also be mentioned, for instance, and it might be nice to have that started here 🙂 It's also slightly more consistent with the other uses, such as `self.get_command()` and `self.get_volume_mounts()`. I'd understand if you do not want to over-engineer at first!